docs/content/concepts/partitions-schedules-sensors/backfills.mdx

```diff
@@ -5,17 +5,17 @@ description: Dagster supports data backfills for each partition or subsets of pa
 
 # Backfills
 
-## Overview
+Backfilling is the process of running partitions for assets that either don’t exist or updating existing records. Dagster supports backfills for each partition or a subset of partitions.
 
-Dagster supports data backfills for each partition or subsets of partitions. After defining a [partitioned asset or job](/concepts/partitions-schedules-sensors/partitions), you can launch a _backfill_ which submits runs to fill in multiple partitions at the same time.
+After defining a [partitioned asset or job](/concepts/partitions-schedules-sensors/partitions), you can launch a backfill that will submit runs to fill in multiple partitions at the same time.
 
----
+Backfills are common when setting up a pipeline for the first time. The assets you want to materialize might have historical data that needs to be materialized to get the assets up to date. Another common reason to run a backfill is when you’ve changed the logic for an asset and need to update historical data with the new logic.
 
-## Launching Backfills
+---
 
-### Partitioned Asset Backfills
+## Launching backfills for partitioned assets
 
-You can open the backfill modal to launch backfills for a partitioned asset using the "Materialize" button, either on the [Asset detail page](https://docs.dagster.io/concepts/partitions-schedules-sensors/partitions#materializing-partitioned-assets) or when viewing a graph of assets. Backfills can also be launched for a selection of differently partitioned assets, as long as the roots share the same partitioning.
+To launch backfills for a partitioned asset, click the **Materialize** button on either the [**Asset details**](/concepts/partitions-schedules-sensors/partitions#materializing-partitioned-assets) or the **Global asset lineage** page. The backfill modal will display. Backfills can also be launched for a selection of differently partitioned assets as long as the roots share the same partitioning.
```

Comment:
I know this was the wording from the original page, but I think "Backfills can also be launched for a selection of differently partitioned assets as long as the roots share the same partitioning." could be made more clear. to be honest i'm not 100% sure what it's trying to say

Reply:
I agree - it's definitely weird. Is this what it's trying to say?:

> Assets that have different partitions can be backfilled together if they use the same partition definition

Or

> Assets that use the same type of partitioning (ex: date) can be backfilled together

Or neither?

Reply:
i think it might be if the most upstream assets in the selection all use the same partition definition. 

I think @clairelin135  might know this off the top of her head. if not i'll put together a little experiment to understand when assets can be backfilled together

Reply:
> most upstream assets in the selection all use the same partition definition

Yup, this is the case.

There's a slight nuance that its the most upstream partitioned assets, but we probably don't need to mention that part

Reply:
cool, thank you! with that knowledge, something like this 

"Backfills can also be launched for a selection of partitioned assets as long as the most upstream assets share the same partitioning, for example: all use a `DailyPartitionsDefinition`" 

makes more sense to me. I think we can drop the "differently partitioned" assets from "selection of differently partitioned assets" since a group of assets with the same partitioning would also follow the "the most upstream assets have to have the same partitioning" rule, and "differently partitioned" introduces one more thing to think about.

Reply:
Agreed - this is much better. Thanks folks!